ide-integration.md

```diff
@@ -30,14 +30,32 @@ Alternatively, you may be able to get tighter integration with a direnv extensio
 
 ## Xcode
 
-Xcode projects can system commands from script build phases and schemes. Since Xcode doesn't source your shell profile the tool versions pinned by your project's Mise configuration file are not automatically resolved through the `$PATH` environment variable.
-To work around this, you can use the `mise x`:
+Xcode projects can run system commands from script build phases and schemes. Since Xcode sandboxes the execution of the script using the tool `/usr/bin/sandbox-exec`, don't expect Mise and the automatically-activated tools to work out of the box. First, you'll need to add `$(SRCROOT)/.mise.toml` to the list of **Input files**. This is necessary for Xcode to allow reads to that file. Then, you'll need to run your commands through the `mise x` command, which 
```

Comment:
I'm not sure if you were going to say more or if the `, which` part should just be removed